package.json

```diff
@@ -1,65 +1,87 @@
 {
```

Comment:
The white-space change in this file makes it difficult to see the functional changes. I know we discussed getting a prettier pre-commit hook for future, but maybe you could fix this one before merge?

Reply:
I can totally do that. As a possible interum solution, could you see if the `hide whitespace changes` option in diff settings helps at all? Sorry for all the white space/formatting changes.

![image](https://user-images.githubusercontent.com/4410245/43280427-d2830d20-90de-11e8-97b5-a89dca91be64.png)